modules/pel/peltool/peltool.py

```diff
@@ -369,6 +393,8 @@ def main():
         parser.add_argument('-l', '--list',
                             help='List PELs',
                             action='store_true')
+        parser.add_argument('-i', '--id', 
+                            dest='idPEL', help='Display a PEL based on its ID')
```

Comment:
I would mention id need to be in hex format if we are not supporting other than hex format.

and also we discussed to use `metavar` for user inputs, right? are you planned to change everything together?

Reply:
Yeah, was thinking to add those details in a single commit in future.